### PR TITLE
Adding annotations and did some clean up.

### PR DESCRIPTION
**Pull request by BigBlueHat**
- Explain why test is skipped.
- Remove skipped test/req no longer in spec.
- Enable createVp in context property test.
- Link to text for section 1.3 Conformance tests.
- Link to text for section 4.3 Identifiers tests.
- Link to text for section 4.3 Types tests.
- Add note about not testing 4.5 name/desc.
- Link to text for section 4.6 Cred Subject tests.
- Link to text for section 4.7 Issuers tests.
- Link to text for section 4.8 Validity Period.
- Link to text for section 4.9 Securing Mechanisms.
- Link to text for section 4.10 Status.

### CODE REVIEW

**Comment by BigBlueHat** [comment]
@aljones15 @tminard I've been working my way through the spec adding these annotations to the code. They should be mostly up-to-date.

However, I'm wanting to pivot off of this baseline and refactor this test suite code a bit. It's currently painful to navigate it as the entire list of requirements is in on big bucket... So, once this is in, I plan to refactor, then return to annotating and sketching out what tests are missing.

File: package.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
     "chai": "^4.3.6",
     "klona": "^2.0.5",
     "mocha": "^10.0.0",
-    "vc-test-suite-implementations": "github:w3c/vc-test-suite-implementations"
+    "vc-test-suite-implementations": "../vc-test-suite-implementations"
```
Comment (aljones15) [inline package.json:48]: this is odd, why are you installing implementations from a local dir?

File: tests/10-vcdm2.js [hunk 2]
```diff
@@ -310,6 +338,7 @@ describe('Verifiable Credentials Data Model v2.0', function() {
         await assert.rejects(endpoints.issue(require(
           './input/credential-issuer-name-extra-prop-en-fail.json')));
       });
+      // TODO: remove. no longer present.
```
Comment (aljones15) [inline tests/10-vcdm2.js:341]: looks like the title just changed: https://w3c.github.io/vc-data-model/#names-and-descriptions

File: tests/10-vcdm2.js [hunk 3]
```diff
@@ -294,6 +321,7 @@ describe('Verifiable Credentials Data Model v2.0', function() {
         await assert.rejects(endpoints.issue(require(
           './input/credential-issuer-object-id-not-url-fail.json')));
       });
+      // TODO: remove. no longer present.
```
Comment (aljones15) [inline tests/10-vcdm2.js:324]: the text of the normative statement just changed: https://w3c.github.io/vc-data-model/#names-and-descriptions

**Review by aljones15 — CHANGES_REQUESTED** [review]
looks like 2 title changes are needed. I do like how you added the google highlight links to each statement before getting to work on the actual test vectors.

File: package.json (hunk 1 above)
Comment (BigBlueHat) [reply package.json:48]: Sorry...yeah. That wasn't supposed to sneak in there. My bad.

File: tests/10-vcdm2.js (hunk 3 above)
Comment (BigBlueHat) [reply tests/10-vcdm2.js:324]: Thanks. Noted.

**Comment by BigBlueHat** [comment]
@aljones15 if you're OK with this, I'd like to wait on doing the `issuer.name` and `issuer.description` test changes. I can make them part of #74 if you'd like--and likely move them into the currently skipped name and description test section. We can then discuss with others where else to run the `name` and `description` tests. Issuer seems like a good candidate (and they're already written), but we can certainly discuss testing them in `credentialSubject` as well (if/when present, of course).

But...I'd love to get this one merged before dealing with that larger move (and base it on #74's new groups).

**Review by aljones15 — APPROVED** [review]
Approving as PR has a lot of really good improvements and comments in it =)

**Pull request merged**